lib/tutorials/en_US/cookies.md

```diff
@@ -53,6 +53,19 @@ The `state()` method is also available on the [response object](/api#response-ob
 return h.response('Hello').state('data', { firstVisit: false });
 ```
 
+## Getting a cookie value
+
+Access a cookie’s value via `request.state` in a route handler, pre-requisite, or request lifecycle extension point.
+
+The `request.state` object contains the parsed HTTP state. Each key represents the cookie name and its value is the defined content.
+
+```javascript
+const value = request.state.data
+// console.log(value) will give you { firstVisit : false }
+```
+
+The sample code uses the `data` cookie key from above where the related value is set to `{ firstVisit: false }`.
+
```

Comment:
I believe the "Overriding options" subsection below is intended to go below the "Setting a cookie" section.

Reply:
Yep, you’re right.